external-import/ransomwarelive/src/lib/ransomConn.py

```diff
@@ -463,11 +472,18 @@ def stix_object_generator(self, item, group_data):
                 external_references.append(external_reference)
 
         # Creating Report object
+        report_name = (
+            item.get("group_name")
+            + " has published a new victim: "
+            + item.get("post_title")
+        )
+        report_published = datetime.strptime(
+            item.get("published"), "%Y-%m-%d %H:%M:%S.%f"
+        )
```

Comment:
```suggestion
        report_published = datetime.fromisoformat(item.get("published"))
```
https://docs.python.org/3.7/library/datetime.html#datetime.datetime.fromisoformat